pkg/provider/framework_provider.go

```diff
@@ -40,6 +40,9 @@ type ProviderConfig struct {
 	OncallAccessToken types.String `tfsdk:"oncall_access_token"`
 	OncallURL         types.String `tfsdk:"oncall_url"`
 
+	ConnectionsAccessToken types.String `tfsdk:"connections_access_token"`
+	ConnectionsURL         types.String `tfsdk:"connections_url"`
```

Comment:
Wonder if it's worth calling these `ConnectionsAPI...` instead of just `Connections` which may be slightly vague?